boards/arm/control_freak/control_freak.dts

```diff
@@ -314,7 +314,7 @@
 		compatible = "jedec,spi-nor";
 		reg = <0>;
 
-		spi-max-frequency = <(10 * 1000 * 1000)>;
+		spi-max-frequency = <DT_FREQ_M(104)>;
```

Comment:
how fast does it actually get can you tell?

Reply:
Don't have easy access to my Saleae for the next few days so won't be able to check this until then. Can still access the filesystem through the shell though and ethernet doesn't seem to be an issue as well. My theory is it'll be the CPU frequency though without any frequency dividers in play